.github/workflows/pr-pull.yaml

```diff
@@ -0,0 +1,20 @@
+name: Pull Request
+run-name: ${{ github.head_ref }} pull request run 🚀
```

Comment:
Other run-name values appear to be Title Case. Where does this appear?

Reply:
That appears as the title of the workflow run in the GitHub action tab.